lib/init/npm-utils.js

```diff
@@ -46,11 +46,13 @@ function findPackageJson(startDir) {
 /**
  * Install node modules synchronously and save to devDependencies in package.json
  * @param   {string|string[]} packages Node module or modules to install
+ * @param   {string} packageManager Package manager to use for installation.
  * @returns {void}
  */
-function installSyncSaveDev(packages) {
+function installSyncSaveDev(packages, packageManager = "npm") {
     const packageList = Array.isArray(packages) ? packages : [packages];
-    const npmProcess = spawn.sync("npm", ["i", "--save-dev"].concat(packageList),
+    const installCmd = packageManager === "yarn" ? "add" : "install";
+    const npmProcess = spawn.sync(packageManager, [installCmd, "-D"].concat(packageList),
```

Comment:
Can we rename this variable, since it doesn't have to be npm? Maybe `installProcess`?

Also, there's a hard-coded "Could not execute npm..." text below, it would be nice to replace `npm` with `${packageManager}`

Reply:
fixed